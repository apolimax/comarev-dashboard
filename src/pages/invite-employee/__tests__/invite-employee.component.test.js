import wrapper from 'test/test-utils';
import InviteEmployee from '../invite-employee.component';

describe('<InviteEmployee />', () => {
  it('renders correctly', () => {
    const rendered = wrapper(InviteEmployee);
    expect(rendered.container).toMatchSnapshot();
  });
});
