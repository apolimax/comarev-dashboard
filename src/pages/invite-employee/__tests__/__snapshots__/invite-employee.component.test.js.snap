// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InviteEmployee /> renders correctly 1`] = `
<div>
  <div
    class="Toastify"
  />
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-appBar-6 MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root sc-eCYdqJ jOPxgf"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <h6
          class="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
        >
          Olá, 
          
        </h6>
        <div>
          <button
            aria-controls="menu-appbar"
            aria-haspopup="true"
            aria-label="account of current user"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </header>
    <div
      class="MuiDrawer-root MuiDrawer-docked makeStyles-drawer-8"
    >
      <div
        class="MuiPaper-root MuiDrawer-paper makeStyles-drawerPaper-9 MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft MuiPaper-elevation0"
      >
        <div />
        <hr
          class="MuiDivider-root"
        />
        <div
          class="makeStyles-list-4"
          role="presentation"
        >
          <ul
            class="MuiList-root MuiList-padding"
          >
            <div
              aria-disabled="false"
              class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                >
                  Início
                </span>
              </div>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
          </ul>
        </div>
        <hr
          class="MuiDivider-root"
        />
      </div>
    </div>
    <main
      class="makeStyles-content-11"
    >
      <div />
      <div
        class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiBox-root MuiBox-root-12"
        >
          <main
            class="MuiContainer-root MuiContainer-maxWidthXs"
          >
            <div
              class="sc-bczRLJ krasoT"
            >
              <h1
                class="MuiTypography-root MuiTypography-h5"
              >
                Convidar novo empregado
              </h1>
              <form
                class="sc-gsnTZi dpoflo"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
                >
                  <label
                    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                    data-shrink="false"
                    for="email"
                    id="email-label"
                  >
                    Endereço de Email
                  </label>
                  <div
                    class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiInput-input"
                      id="email"
                      name="email"
                      type="text"
                      value=""
                    />
                  </div>
                  
                </div>
                <button
                  aria-label="Enviar e-mail"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained sc-dkzDqf cskwdh MuiButton-containedPrimary MuiButton-fullWidth"
                  tabindex="0"
                  type="submit"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Enviar
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </form>
            </div>
          </main>
        </div>
      </div>
    </main>
  </div>
</div>
`;
